controllers/neutronapi_controller.go

```diff
@@ -412,12 +412,18 @@ func (r *NeutronAPIReconciler) reconcileInit(
 		ports[metallbcfg.Endpoint] = portCfg
 	}
 
+	routeAnnotations := make(map[string]string)
+	if instance.Spec.RouteAnnotations.Timeout != "" {
```

Comment:
I would have liked to do this with a range but couldn't find an example in the other operators I checked.

Reply:
I think Lewis is referring to enumerating the struct. But maybe the better option is as you already said, to just use map[string]string so that we aren’t explicit about the annotation provided.